CHANGES.md

```diff
@@ -10,10 +10,14 @@ To be released.
 
 ### Added interfaces
 
+- Added IPv6 support in `Libplanet.Stun.StunAddress` [[#267]]
```

Comment:
I believe this should go to <q>Behavioral changes</q> section.

```suggestion
- Added IPv6 support to `Libplanet.Stun.StunAddress`.  [[#267], [#271]]
```

Reply:
I moved it.